portfolio/src/main/webapp/locations.js

```diff
@@ -0,0 +1,68 @@
+var locations = [
```

Comment:
I'm not quite familiar with JS, according to here: https://google.github.io/styleguide/jsguide.html#appendices-legacy-exceptions-var, do we need a @const annotation?

Reply:
changed `var` to `const`